otlp/logs_test.go

```diff
@@ -25,52 +25,7 @@ func TestTranslateLogsRequest(t *testing.T) {
 
 	testServiceName := "my-service"
 
-	req := &collectorlogs.ExportLogsServiceRequest{
-		ResourceLogs: []*logs.ResourceLogs{{
-			Resource: &resource.Resource{
-				Attributes: []*common.KeyValue{{
-					Key: "resource_attr",
-					Value: &common.AnyValue{
-						Value: &common.AnyValue_StringValue{StringValue: "resource_attr_val"},
-					},
-				}, {
-					Key: "service.name",
-					Value: &common.AnyValue{
-						Value: &common.AnyValue_StringValue{StringValue: testServiceName},
-					},
-				}},
-			},
-			ScopeLogs: []*logs.ScopeLogs{{
-				Scope: &common.InstrumentationScope{
-					Name:    "library-name",
```

Comment:
Hmmm. In the refactor, this is changed to be `instr_scope_name` and `instr_scope_version`. But this test case is inconsistent with the one that follows it. Any ideas?

Reply:
Yup. We talked about it in a Zoom: this string is set only for the expectation for the value of `library.name` attributes on the resulting event. It can be whatever the test should expect, value is arbitrary so long as what goes in is what comes out.